.Lib9c.Tools/SubCommand/Tx.cs

```diff
@@ -0,0 +1,104 @@
+using Bencodex;
+using Bencodex.Types;
+using Cocona;
+using Libplanet;
+using Libplanet.Assets;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Libplanet.Tx;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace Lib9c.Tools.SubCommand
+{
+    public class Tx
+    {
+        private static Codec _codec = new Codec();
+
+        [Command(Description = "Create TransferAsset action and dump it.")]
+        public void TransferAsset(
+            [Argument("SENDER", Description = "An address of sender.")] string sender,
+            [Argument("RECIPIENT", Description = "An address of recipient.")] string recipient,
+            [Argument("AMOUNT", Description = "An amount of gold to transfer.")] int goldAmount,
+            [Argument("GENESIS-BLOCK", Description = "Specifies the path of the genesis block.")] string genesisBlock
```

Comment:
I'm not sure how to describe is better ([comment](https://github.com/planetarium/lib9c/pull/125/files#r501663596)). But I believe it will be better to describe it as a noun for consistency at least.

Reply:
🆗